### PR TITLE
SSCS-10286 Critical Data Points For Hearing Journey

### PR DESCRIPTION
**Pull request by TeaWithLucas**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-10286

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/CancellationReason.java [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
+    STRUCK_OUT("BBA3-struckOut", "Struck Out"),
+    WITHDRAWN("BBA3-withdrawn", "Withdrawn");
 
     private final String hmcReference;
-    private final String value;
+    private final String label;
+
+    public static CancellationReason getCancellationReasonByHmcReference(String value) {
+        return Arrays.stream(CancellationReason.values())
+                .filter(sl -> sl.getHmcReference().equals(value))
```
Comment (wmarshid) [inline src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/CancellationReason.java:26]: optional: You know already I don't like this lol.. what is `sl`? maybe change this to cancellationReason? and one below too.

**Review by wmarshid — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/CancellationReason.java (hunk 1 above)
Comment (TeaWithLucas) [reply src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/CancellationReason.java:26]: Will do

**Review by johnoneill46 — APPROVED** [review]

**Review by wmarshid — CHANGES_REQUESTED** [review]
Do this instead please - https://github.com/hmcts/sscs-common/pull/1196
Let me know if you want to discuss.

File: src/test/java/uk/gov/hmcts/reform/sscs/service/VenueDataLoaderTest.java [hunk 2]
```diff
@@ -7,18 +7,46 @@
...
+
+    private VenueDetails.VenueDetailsBuilder venueDetailsBuilder;
+
+
+    @Before
+    public void setUp() {
+        venueDataLoader = new VenueDataLoader();
         venueDataLoader.init();
+
+        venueDetailsBuilder = VenueDetails.builder()
```
Comment (wmarshid) [inline src/test/java/uk/gov/hmcts/reform/sscs/service/VenueDataLoaderTest.java:33]: minor: don't think we need this anymore?

File: src/test/java/uk/gov/hmcts/reform/sscs/service/VenueDataLoaderTest.java [hunk 3]
```diff
@@ -7,18 +7,46 @@
...
 import org.junit.runner.RunWith;
 import uk.gov.hmcts.reform.sscs.ccd.domain.Address;
 import uk.gov.hmcts.reform.sscs.ccd.domain.Venue;
+import uk.gov.hmcts.reform.sscs.model.VenueDetails;
 
 @RunWith(JUnitParamsRunner.class)
 public class VenueDataLoaderTest {
 
-    private static final VenueDataLoader venueDataLoader = new VenueDataLoader();
+    private static final String VALID_EPIMS_ID = "45900";
```
Comment (wmarshid) [inline src/test/java/uk/gov/hmcts/reform/sscs/service/VenueDataLoaderTest.java:21]: minor: would prefer to move this into the method if `venueDetailsBuilder` is no longer required

**Review by wmarshid — APPROVED** [review]
LGTM 👌  just two minor comments

**Pull request merged**